doc/manual.md

```diff
@@ -358,21 +358,40 @@ local x1 : integer = ns[1]
 local x2 : integer = ns[4] -- run-time error
 ```
 
+## Exporting top-level components
+
+The `export` keyword can be used to export top-level components such as variables
+and functions from a Pallene module. The keyword `export` can be used in the
+same places where the `local` keyword can be used. It should be noted that, every
+top-level component declaration should have either `local` or `export` modifier.
+
+In the following example, we export `foo` and `name` from our module.
+```lua
+export function foo(x: integer): integer
+    return x + 1
+end
+
+export name: string = "Hello"
+```
+
+You can export on top-level components from a Pallene module. Further, it is an
+error to export the same name twice.
+
 ## The Complete Syntax of Pallene
 
 Here is the complete syntax of Pallene in extended BNF.
 As usual, {A} means 0 or more As, and \[A\] means an optional A.
 
-    program ::= {toplevelrecord | topleveltypealias | toplevelvar | toplevelfunc}
+    program ::= {top-levelrecord | top-leveltypealias | top-levelvar | top-levelfunc}
```

Comment:
I'd prefer if the names of the non-terminals matched the ones that we are using in parser.lua. Unfortunately, I don't think we can use hyphens there.

Reply:
I accidentally replaced toplevel with top-level in the grammar.

Reply:
Yeah... Let's keep using `toplevel` everywhere.

Reply:
By the way, please add the new export keyword to the "Complete Syntax of Pallene" section in the manual.